net/Kconfig

```diff
@@ -117,6 +117,12 @@ config NET_GUARDSIZE
 		packet size will be chopped down to the size indicated in the TCP
 		header.
 
+config NET_RECV_BUFSIZE
+	int "Net Receive buffer size"
+	default 65535
```

Comment:
should we change to zero? so the default setting is same as before(don't limit the receive buffer size).